docs/proposals/user-namespace.md

```diff
@@ -0,0 +1,90 @@
+# Support Host User Namespaces
+
+## Motivation
+
+Docker 1.10 supports [user namespace remapping](https://docs.docker.com/v1.10/engine/reference/commandline/daemon/#daemon-user-namespace-options)
+in the docker daemon.  This allows you to run as the root user in a 
+container while running as a non-root user in the host.
+
+Additionally, the `HostConfig` for a container now contains a parameter
+that lets you escape the private user namespace and indicate that you
+would like to [use the host's user namespace](https://github.com/docker/engine-api/blob/master/types/container/host_config.go#L310). 
+
+If the Docker daemon is launched with the remap option in a Kubernetes 
+cluster today and a pod author would like to run a container that uses 
+the host user namespace there is not currently a way to express the 
+configuration.
+
+## Assumptions
+
+1.  There is currently a bug in Docker that does not allow you to use
+`--net=host`, `--pid=host`, or `--ipc=host` when remapping is enabled.
+This is fixed in https://github.com/docker/docker/pull/25771 such that
+if you are enabling user namespace remapping you may use `--net=host`, `--pid=host`, 
+and `--ipc=host` flags if `--userns=host`.
+
+## Use Cases
+
+Assuming that the docker daemon is started with --userns-remap=default, 
```

Comment:
You should start this with "Kubernetes cluster administrators wish to have a higher level of default cluster security on Docker 1.10+ by enabling user namespace remapping, but want to be able to continue to run cluster level actions for monitoring."